modules/snooker/src/main/ts/ephox/snooker/model/GridRow.ts

```diff
@@ -3,6 +3,9 @@ import { SugarElement } from '@ephox/sugar';
 
 import * as Structs from '../api/Structs';
 
+type RowMorphism = (element: SugarElement<HTMLTableRowElement | HTMLTableColElement>) => SugarElement<HTMLTableRowElement | HTMLTableColElement>;
```

Comment:
Should `RowMorphism` have the `index: number` argument as well for parity?

Reply:
No, as within the context of this module it'll only ever convert one row and not an array like is the case with cells. Maybe we need to rename it to be something else? I'm open to suggestions as I really didn't know what to call these 2.

Reply:
I'm okay with this name I think — if someone else has a suggestion though that's fair